### PR TITLE
docs: add CONTRIBUTING and CONTRIBUTORS section in README.md

### PR DESCRIPTION
**Pull request by smoya**
**Description**

This PR adds:

- a link to [CONTRIBUTING](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md).
- a badge with the number of contributors (via `all-contributors`) in `README.md`
- all project contributors  (via `all-contributors`)  in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <h5 align="center">
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```
Comment (jonaslagoni) [inline README.md:4]: Not rendered accurately 🙂

**Review by jonaslagoni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (smoya) [reply README.md:4]: It's true :O I used `all-contributors` CLI to generate this :suspect: . I'm gonna fix it.

Comment (smoya) [reply README.md]: Fixed!

Comment (jonaslagoni) [reply README.md]: Yea, they have a problem with the badge, you cant have it inline together with other badges either 😅

**Review by jonaslagoni — APPROVED** [review]
👍

**Comment by smoya** [comment]
@all-contributors please add @jonaslagoni for review

**Pull request merged**